sass/easyopac_status.scss

```diff
@@ -26,17 +52,63 @@
       .close-button {
         position: relative;
         display: block;
-        &::after {
-          font-family: 'icomoon';
+        &:before {
           font-size: 54px;
-          line-height: 1;
-          color: rgba(38,38,38,.54);
-          display: block;
-          position: absolute;
-          top: 0;
           content: "\e907";
+          font-family: 'icomoon';
         }
       }
     }
   }
+  .critical {
```

Comment:
Why not move `font-size: 55px;` and `font-family: 'icomoon';` to generic selector instead of defining it 5 times?